samples/SmokeTest/BlobStorage.ts

```diff
@@ -0,0 +1,51 @@
+import { SharedKeyCredential, BlobServiceClient, ContainerClient } from "@azure/storage-blob";
+
+export class BlobStorage{
+
+    private static ContainerClient : ContainerClient;
+    private static blobName : string;
+
+    static async Run(){
+        console.log();
+        console.log("------------------------");
+        console.log("Storage - Blobs");
+        console.log("------------------------");
+        console.log("1) Upload Blob");
+        console.log("2) Delete Blob (Clean up the resource)");
+        console.log();
+
+        const account = process.env["STORAGE_ACCOUNT_NAME"]     || "<YourStorageAccountNAME>";
+        const accountKey = process.env["STORAGE_ACCOUNT_KEY"]   || "<YourStorageAccountKEY>";
+        const containerName = "mycontainer";
+        BlobStorage.blobName = "JSNewBlob";
+
+        const credential = new SharedKeyCredential(account, accountKey);
+        const serviceClient = new BlobServiceClient(`https://${account}.blob.core.windows.net`,credential);
+        BlobStorage.ContainerClient = serviceClient.getContainerClient(containerName);
+
+        await BlobStorage.UploadBlob();
+        await BlobStorage.CleanUp();
```

Comment:
Might consider wrapping the UploadBlob in a try/catch and calling `CleanUp` in a finally block.